e2e/cypress/test_module/cypress/support/step_definitions/products-page.steps.ts

```diff
@@ -0,0 +1,108 @@
+/// <reference types="Cypress" />
+
+import { Given, Then, When } from "@badeball/cypress-cucumber-preprocessor";
+import { sortingNameParamMap } from "@cypress-e2e/fixtures/constants";
+
+Given("The user is on the products page", () => {
+  cy.visit("http://localhost:3000/products");
+  cy.intercept("GET", /\/api\/v1\/products/).as("productsPageRequest");
+});
+
+Given("The user is on the first page of products", () => {
+  cy.visit("http://localhost:3000/products?page=1");
+  cy.intercept("GET", /\/api\/v1\/products/).as("productsPageRequest");
+});
+
+Given("The user is on the second page of products", () => {
+  cy.visit("http://localhost:3000/products?page=2");
+  cy.intercept("GET", /\/api\/v1\/products/).as("productsPageRequest");
```

Comment:
We have global-data.ts with constants httpMethod. You should use them

Reply:
Thank you, fixed